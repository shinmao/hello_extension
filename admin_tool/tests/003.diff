001+ Fatal error: Uncaught Error: Call to undefined function admin_tool_test2() in /var/www/html/so/admin_tool/tests/003.php:2
002+ Stack trace:
001- string(11) "Hello World"
002- string(9) "Hello PHP"
003+ #0 {main}
004+   thrown in /var/www/html/so/admin_tool/tests/003.php on line 2