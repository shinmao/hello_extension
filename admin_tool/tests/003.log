
---- EXPECTED OUTPUT
string(11) "Hello World"
string(9) "Hello PHP"
---- ACTUAL OUTPUT
Fatal error: Uncaught Error: Call to undefined function admin_tool_test2() in /var/www/html/so/admin_tool/tests/003.php:2
Stack trace:
#0 {main}
  thrown in /var/www/html/so/admin_tool/tests/003.php on line 2
---- FAILED
