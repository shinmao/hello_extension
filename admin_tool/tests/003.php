<?php
var_dump(admin_tool_test2());
var_dump(admin_tool_test2('PHP'));
?>
