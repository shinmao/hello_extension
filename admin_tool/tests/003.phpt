--TEST--
admin_tool_test2() Basic test
--SKIPIF--
<?php
if (!extension_loaded('admin_tool')) {
	echo 'skip';
}
?>
--FILE--
<?php
var_dump(admin_tool_test2());
var_dump(admin_tool_test2('PHP'));
?>
--EXPECT--
string(11) "Hello World"
string(9) "Hello PHP"
