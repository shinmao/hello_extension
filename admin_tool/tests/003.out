Fatal error: Uncaught Error: Call to undefined function admin_tool_test2() in /var/www/html/so/admin_tool/tests/003.php:2
Stack trace:
#0 {main}
  thrown in /var/www/html/so/admin_tool/tests/003.php on line 2